src/v2/Apps/Consign/Routes/SubmissionFlow/UploadPhotos/Components/UploadPhotosForm.tsx

```diff
@@ -97,6 +112,13 @@ export const UploadPhotosForm: React.FC<UploadPhotosFormProps> = ({
 
   return (
     <Box {...rest}>
+      <ErrorModal
+        show={showErrorModal}
+        headerText="An error occurred"
+        contactEmail="orders@artsy.net"
```

Comment:
the email address might change as per product requirements.

Reply:
Thanks for pointing this out. I see that this is being discussed in the [Jira ticket](https://artsyproduct.atlassian.net/browse/SWA-49?focusedCommentId=43152), so let's keep discussing in that thread before merging this ticket.

Would you mind making this a WIP PR until we resolve this?